### PR TITLE
Add Cryptography

### PR DESCRIPTION
**Pull request by rosdyana**
Add cryptography, currently support md5, sha1, is there any list that I can add more hashing/cryptography ?

### CODE REVIEW

**Comment by ashwini0529** [comment]
Hey,
Thank you for your PR. 🎉 
Maybe we can add more features to the same. For example, Bcrypt, Whirpool etc.
@shubhodeep9 @akshitgrover suggestions?

**Comment by rosdyana** [comment]
Hi, I added Bcrypt and Whirlpool.
Cheers!! :tada:

File: hackr/crypto.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    def __init__(self, plain_text):
+        self.plain_text = plain_text
+
+    def sha1(self):
+        encrypted = hashlib.sha1(self.plain_text).hexdigest()
+        print encrypted
+
+    def md5(self):
+        encrypted = hashlib.md5(self.plain_text).hexdigest()
+        print encrypted
```
Comment (ashwini0529) [inline hackr/crypto.py]: `return` instead of that?
Same for other functions as well.

**Review by ashwini0529 — CHANGES_REQUESTED** [review]

File: hackr/crypto.py (hunk 1 above)
Comment (rosdyana) [reply hackr/crypto.py]: sure, thanks for the correction :tada:

**Comment by ashwini0529** [comment]
@rosdyana Thank you for your PR! 🎉

**Pull request merged**